tests/validate/bandwidth_test/src/host.cpp

```diff
@@ -80,12 +80,26 @@ int main(int argc, char **argv) {
   // and will return the pointer to file buffer.
   auto fileBuf = xcl::read_binary_file(binaryFile);
   cl::Program::Binaries bins{{fileBuf.data(), fileBuf.size()}};
-  if (dev_id >= devices.size()) {
-    std::cout << "The device_id provided using -d flag is outside the range of "
-                 "available devices\n";
-    return EXIT_FAILURE;
+
+  int found = dev_id.find(":");
```

Comment:
auto pos = dev_id.find(...);   The return value is not an int.

Reply:
updated.